src/express-route-cross-connection/azext_expressroutecrossconnection/custom.py

```diff
@@ -0,0 +1,130 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from ._client_factory import network_client_factory
+
+
+def list_express_route_cross_connections(cmd, resource_group_name=None):
+    client = network_client_factory(cmd.cli_ctx).express_route_cross_connections
+    if resource_group_name:
+        return client.list_by_resource_group(resource_group_name)
+    return client.list()
+
+
+def update_express_route_cross_connection(instance, provisioning_state=None, notes=None, bandwidth_in_mbps=None):
+
+    if bandwidth_in_mbps is not None:
```

Comment:
The provider is not allowed to change the bandwidth for the circuit.

Reply:
@karthikananth from our discussion I was left with the impression that the provider could change this parameter.

Reply:
Karthik clarified and I have removed this parameter.